crates/daphne/src/roles/helper/handle_agg_job.rs

```diff
@@ -21,7 +21,11 @@ pub struct HandleAggJob<S> {
 }
 
 /// The initial state, the aggregation request has been received.
-pub struct Init(DapRequest<AggregationJobInitReq>);
+pub struct Init(DapRequest<HashedAggregationJobReq>);
+
+/// The aggregation job is legal. Which means that it's either new or it's parameters haven't
+/// changed since the last time we've received it.
+pub struct LegalAggregationJobReq(DapRequest<AggregationJobInitReq>);
```

Comment:
Incidentally: we were discussing whether we should use the term "agg job init req" or just "agg job req". The difference matters if we support VDAFs with more than one round of preparation, which we currently don't, and I hope we won't have to.

However, in this particular case I think the name is fine for another reason: the parameters we don't want to change pertain to an entire aggregation job, not just the initial request for that job.